js/ftx.js

```diff
@@ -1565,51 +1565,60 @@ module.exports = class ftx extends Exchange {
             params = this.omit (params, [ 'clientId', 'clientOrderId' ]);
         }
         let method = undefined;
-        const stopPrice = this.safeValue2 (params, 'stopPrice', 'triggerPrice');
-        params = this.omit (params, [ 'stopPrice', 'triggerPrice' ]);
-        if (((type === 'limit') || (type === 'market')) && (stopPrice === undefined)) {
+        let triggerPrice = this.safeString (params, 'triggerPrice');
```

Comment:
Have we lost the support for the `stopPrice` param for backward-compatibility here?